From 31015d2b0a6f7d9a03167f5e759c48d426d6707b Mon Sep 17 00:00:00 2001
From: Pekka Vuorela <pekka.vuorela@jollamobile.com>
Date: Fri, 8 Dec 2017 15:17:37 +0200
Subject: [PATCH] Honor configuration for building glibc copy of strtok_r

---
 goo/glibc_strtok_r.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/goo/glibc_strtok_r.cc b/goo/glibc_strtok_r.cc
index e779bf50..2e088ae5 100644
--- a/goo/glibc_strtok_r.cc
+++ b/goo/glibc_strtok_r.cc
@@ -56,6 +56,8 @@
 //
 //========================================================================
 
+#include "config.h"
+
 #ifndef HAVE_STRTOK_R
 
 #include <string.h>
-- 
2.13.6

